### ISSUE TITLE
compilation breaks after bison 3.7.x upgrade

### ISSUE THREAD

**Issue by priv-kweihmann** [splintchecker/splint#20]
After updating to bison 3.7.1 compilation doesn't work anymore - compiler is complains about syntax errors.
Same files work with versions < 3.7.0

`find -name "*.y" -exec bison --update {} \;` does return

```
./src/mtgrammar.y: warning: 11 shift/reduce conflicts [-Wconflicts-sr]
./src/mtgrammar.y: note: rerun with option '-Wcounterexamples' to generate conflict counterexamples
./src/cgrammar.y: warning: 159 shift/reduce conflicts [-Wconflicts-sr]
./src/cgrammar.y: warning: 123 reduce/reduce conflicts [-Wconflicts-rr]
./src/cgrammar.y: note: rerun with option '-Wcounterexamples' to generate conflict counterexamples
./src/llgrammar.y: warning: 2 shift/reduce conflicts [-Wconflicts-sr]
./src/llgrammar.y: note: rerun with option '-Wcounterexamples' to generate conflict counterexamples
```

Used versions
* bison 3.7.1
* gcc 10.2.0
* splint a28a60fd537cb0271a43a54f1f09870e03c2ba1a

**Comment by millert** [comment]
Bison 3.7 now includes the generated header file so the .c files need to be modified to include the correct file after things get renamed.  This is fixed by #26
